src/videojs.ima.js

```diff
@@ -17,7 +17,22 @@
  * https://www.github.com/googleads/videojs-ima
  */
 
-(function(vjs) {
+ (function (root, factory) {
+   if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module unless amdModuleId is set
+    define(["videojs"], function (a0) {
+
+      return (factory(a0));
+    });
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory(require("video.js"));
+  } else {
+    root['videojs-ima'] = factory(videojs);
+  }
+}(this, function(vjs) {
```

Comment:
Could you expand on the changes above? Are they required for the IE change to work? If not, could you pull them out and (if required) submit them as a separate pull request?